flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -154,6 +203,11 @@ class FlxGamepad implements IFlxDestroyable
 		return gamepadButton;
 	}
 	
+	public inline function getFlipAxis(AxisID:Int):Int
```

Comment:
Good catch on this, as it was also broken on "Adobe Windows" Capabilities.manufacturer value (ie, the standalone flash player on windows).